tests/integration/targets/mso_tenant/tasks/main.yml

```diff
@@ -89,6 +85,7 @@
   - ansible_test2
   - ansible_test3
   - tenant_with_site
+  ignore_errors: true
```

Comment:
Ignoring the error when the `ansible_test` is used by other schema or templates.

Reply:
This need to be taken care off by the cleanup steps we cannot progress from this task if the tenant to be used is not removed.

Remove ignore_errors